barretenberg/cpp/src/barretenberg/vm/avm/generated/recursive_verifier.cpp

```diff
@@ -48,7 +48,7 @@ template <typename Flavor>
 AvmRecursiveVerifier_<Flavor>::AggregationObject AvmRecursiveVerifier_<Flavor>::verify_proof(
     const HonkProof& proof, const std::vector<std::vector<bb::fr>>& public_inputs_vec_nt, AggregationObject agg_obj)
 {
-    StdlibProof<Builder> stdlib_proof = bb::convert_proof_to_witness(builder, proof);
+    StdlibProof<Builder> stdlib_proof = bb::convert_native_proof_to_stdlib(builder, proof);
```

Comment:
Could you also modify bb-pilcom/bb-pil-backend/templates/recursive_verifier.cpp.hbs ? Otherwise this will get overwritten. Thanks!